packages/swap/src/routes/v2/swap.ts

```diff
@@ -0,0 +1,408 @@
+import express from 'express';
+import { assetConstants, getAssetAndChain } from '@/shared/enums';
+import { assertUnreachable, getPriceFromPriceX128 } from '@/shared/functions';
+import { getRequiredBlockConfirmations } from '@/swap/utils/rpc';
+import prisma, { Prisma, Swap, SwapFee } from '../../client';
+import { getPendingBroadcast, getPendingDeposit } from '../../ingress-egress-tracking';
+import { readField } from '../../utils/function';
+import logger from '../../utils/logger';
+import ServiceError from '../../utils/ServiceError';
+import {
+  channelIdRegex,
+  coerceChain,
+  swapRequestId,
+  txHashRegex,
+  estimateSwapDuration,
+  isEgressableSwap,
+  failedSwapMessage,
+} from '../../utils/swap';
+import { asyncHandler } from '../common';
+import { Failure } from '../swap';
+
+const router = express.Router();
+
+export enum StateV2 {
+  Failed = 'FAILED',
+  Complete = 'COMPLETE',
+  Broadcasted = 'BROADCASTED',
```

Comment:
removed broadcast aborted

Reply:
i think egress scheduled can be removed because it always happens in the same block as broadcast requested afaik.

we should reduce this further into maybe more high level waiting, receiving, swapping, sending, sent (ingress/egress tracker), complete. and if we have more information available we can put it into some other substatus i think. probably better to just have broad strokes here

Reply:
yeah we removed `broadcast_requested` because its the same as `egress_scheduled` basically. But we keep broadcasted as that expects the tx in the mempool

Reply:
i'm open to renaming some of these statuses to simpler language. will try my hand at it first.